lib/sentry/client.ex

```diff
@@ -21,7 +21,7 @@ defmodule Sentry.Client do
   """
   @spec send_event(%Event{}) :: {:ok, String.t} | :error
   def send_event(%Event{} = event) do
```

Comment:
I'm not sure that that would be necessary.  This kind of pattern would usually come along with specifying a Behaviour, which might be overkill at this point 🙂